styles/prosilver/template/pastebin_body.html

```diff
@@ -88,5 +66,27 @@
 </div>
 
 
+<!-- IF S_AUTH_VIEW -->
+<div class="mini-panel navbar">
+	<span class="corners-top"><span></span></span>
```

Comment:
Okay das hat jetzt aber in erster Linie nichts mit diesem PR zu tun, diesen Codeteil habe ich aus Designgründen lediglich verschoben. Um die anderen angemerkten Punkte kümmere ich mich, danke.